3.7/graphs-satellite-graphs-management.md

```diff
@@ -0,0 +1,333 @@
+connected later on.
+
+All collections used within the creation process are newly created or ,if already
+available, checked against the collection properties. The process will fail if one
+of them already exists, unless they have the correct sharding already. All newly
+created collections will immediately be dropped again in the failure case.
+
+**Examples**
+
+Create a graph without relations. Edge definitions can be added later:
+
+{% arangoshexample examplevar="examplevar" script="script" result="result" %}
+    @startDocuBlockInline satelliteGraphCreateManagement_1_cluster
+    @EXAMPLE_ARANGOSH_OUTPUT{satelliteGraphCreateManagement_1_cluster}
+    var satelliteGraphModule = require("@arangodb/satellite-graph");
+    var graph = satelliteGraphModule._create("satelliteGraph");
+    satelliteGraphModule._graph("satelliteGraph");
+    ~satelliteGraphModule._drop("satelliteGraph", true);
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock satelliteGraphCreate1_1_cluster
+{% endarangoshexample %}
+{% include arangoshexample.html id=examplevar script=script result=result %}
+
+Create a graph using an edge collection `edges` and a single vertex collection
+`vertices` as relation:
+
+{% arangoshexample examplevar="examplevar" script="script" result="result" %}
+    @startDocuBlockInline satelliteGraphCreateManagement_2_cluster
+    @EXAMPLE_ARANGOSH_OUTPUT{satelliteGraphCreateManagement_2_cluster}
+      var graph_module = require("@arangodb/satellite-graph");
+      var edgeDefinitions = [ graph_module._relation("edges", "vertices", "vertices") ];
+      var graph = graph_module._create("myGraph", edgeDefinitions);
+      graph_module._graph("myGraph");
+     ~graph_module._drop("myGraph", true);
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock satelliteGraphCreateManagement_2_cluster
+{% endarangoshexample %}
+{% include arangoshexample.html id=examplevar script=script result=result %}
+
+Create a graph with edge definitions and orphan collections:
+
+{% arangoshexample examplevar="examplevar" script="script" result="result" %}
+    @startDocuBlockInline satelliteGraphCreateManagement_3_cluster
+    @EXAMPLE_ARANGOSH_OUTPUT{satelliteGraphCreateManagement_3_cluster}
+      var graph_module = require("@arangodb/satellite-graph");
+      var edgeDefinitions = [ graph_module._relation("myRelation", ["male", "female"], ["male", "female"]) ];
+      var graph = graph_module._create("myGraph", edgeDefinitions, ["sessions"]);
+      graph_module._graph("myGraph");
+     ~graph_module._drop("myGraph", true);
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock satelliteGraphCreateManagement_3_cluster
+{% endarangoshexample %}
+{% include arangoshexample.html id=examplevar script=script result=result %}
+
+Modify a SatelliteGraph definition at runtime
+------------------------------------
+
+After you have created a SatelliteGraph its definition is not immutable. You can
+still add or remove relations. This is again identical to General Graphs.
```

Comment:
mhm I wonder we write:
`SatelliteGraph`
and
`General Graphs`

mind the gap.
Is this intentional, which should be the correct writing?
@Simran-B

Reply:
Enterprise Edition features: FooBar (no space), but General Graphs with a space as per Ingo.